GameDefinition/Content/ItemKind.hs

```diff
@@ -1630,16 +1631,18 @@ halberd = ItemKind
   , ikit     = []
   }
 halberd2 = halberd
-  { ifreq    = [("common item", 3 * 2), ("starting weapon", 1)]
+  { iname    = "Pollaxe"
+  , ifreq    = [("common item", 3 * 2), ("starting weapon", 1)]
   , iweight  = 4000
   , idamage  = 18 `d` 1
-  -- , idesc    = ""
+  , idesc    = "A long-hafted axe: once used for maintenance, now turned to a bloodier purpose."
   }
 halberd3 = halberd
-  { ifreq    = [("common item", 1 * 2)]  -- compensating for low base rarity
+  { iname    = "halberd"
+  , ifreq    = [("common item", 1 * 2)]  -- compensating for low base rarity
   , iweight  = 5000
   , idamage  = 24 `d` 1
-  -- , idesc    = ""
+  , idesc    = "Violence is occasionally a regretful necessity; this finely-balanced and modernised version of an ancient weapon ensures it is a brief one."
```

Comment:
I struggled with this one. I wanted a weapon that seemed more deliberate, to justify its power relative to the other halberd types. What I thought might work was a modern anti-boarding weapon - in a future with frequent combat on ships, you'd expect such weapons to exist, although be rarely used. However, I struggled to put that concept into a sufficiently concise form.

Reply:
Actually, you have a point. That's a tremendously powerful melee weapon, and so it's rather unlikely in the near future (with a gun there would be no problem to justify it's power, only the risk of decompression would require care and would explain its rarity). I wouldn't mind removing it altogether. And we are already straining suspension of disbelief by letting heroes discover many powerful artifact melee weapons out of museum, personal heirloom stashes, etc.

OTOH, it never occurred to me that the decompression conundrum of near future space faring could be solved by resurrecting and perfecting some melee weapons. I used decompression as justification of no weapons at all, but obviously populous ships require some kind of weapons for the officers at least. I rather don't envision piracy at this point, mostly because sadly our heroes can't engage in it, nor in any other space battles (though I'd love to add it to the game in 10 or 20 years from now and then your weapon would fit). 

But indeed some actual non-lethal weapons are realistic, e.g., tazer, glue launcher, pepper gel or some other chemical agent, flashbang grenade, as well as some lethal ones that don't puncture ship hulls, e.g., pistols with expanding bullets, shotguns with special ammo. I still haven't decided how to implement launchers (bows, guns) and I want to keep them rare at least until endgame to keep melee fun, but perhaps the idea is ripe now and we could open an issue and start brainstorming.

Reply:
And if you decide to remove the `24 d 1` item, I think it makes sense to double the frequency of the `18 d 1`. it is very rare anyway. Currently seeing any weapon with increased damage dice occurs once in several won games. They are rarer than unique artifacts, and in fact sometimes more powerful, but in a simple way (just higher kinetic damage). In the future, with much larger spaceship (40 levels, with 2 to 5 caves per level), I'd aim at once per game that explores half the caves, on average at most.